.gitignore

```diff
@@ -3,4 +3,8 @@
 .*
 !.git*
 /imgui*
+/game/cpml*
+/game/dkjson.lua
+/game/lux*
+/game/steaming*
```

Comment:
who is responsible for this???? this is 10000% unnecessary, and already resolved by putting everything in the `game/lib/` dir, stop this immediately

Reply:
@rilifon probably did this after noticing the files appered as untracked (instead of ignored) by git. This happened because we moved the files before and he probably didn't run `make clean`, thus  leaving the obsolete files in their old locatiom in additiom to the correct one.

All in all, I think it's very understandable that he did this. Although we do have to remove it. Will do tomorrow.

Reply:
@Kazuo256 comments explains my line of thought